templates/person/show.json.twig

```diff
@@ -0,0 +1,71 @@
+{
+    "@context": "https://schema.org",
+    "@graph": [
+        {
+            "@type": "Person"
+            {% if person.firstname %},"givenName": "{{ person.firstName}}" {% endif %}
+            {% if person.lastName %},"familyName": "{{ person.lastName}}" {% endif %}
+            ,"gender": "{{ person.gender }}"
```

Comment:
For gender, you should use the TWIG gender partial (`templates/person/gender.html.twig`) since it provides a consistent series of captions. You can do that by using the include tag (https://twig.symfony.com/doc/3.x/tags/include.html):

```twig
"gender": {% include 'person/gender.html.twig' %}
```